extensions/blocks/contact-form/index.js

```diff
@@ -56,6 +56,22 @@ export const settings = {
 			type: 'string',
 			default: null,
 		},
+		customThankyou: {
```

Comment:
The other attributes are snake case, but I don't know if that's just for legacy reasons...

Reply:
It looks like the snake case attributes are legacy from the shortcode, `hasFormSettingsSet` and `submitButtonText` appear to be the block replacements for `has_form_settings_set` and `submit_button_text`, respectively.